docs/specifications.md

```diff
@@ -0,0 +1,18 @@
+## Quiz-Projekt
+
+# Lastenheft
+
+
+Einzelspieler
```

Comment:
Should use items (`*`) because subsequent lines without a newline in between are merged into a single line.